server/api.js

```diff
@@ -23,7 +23,7 @@ const db = require('./db/db')
 const paApiKeys = [helpers.getEnvVar('PA_API_KEY_PRIMARY'), helpers.getEnvVar('PA_API_KEY_SECONDARY')]
 
 async function authorize(req, res, next) {
-  if (paApiKeys.indexOf(req.query.braveKey) < 0 && paApiKeys.indexOf(req.body.braveKey)) {
+  if (paApiKeys.indexOf(req.query.braveKey) < 0 && paApiKeys.indexOf(req.body.braveKey) < 0) {
```

Comment:
Thanks for catching this and pointing it out!

You have no idea how happy I am that other people are looking at my code and catching things!

Reply:
😊